yasa/hypno.py

```diff
@@ -31,15 +31,18 @@
 import logging
 import numpy as np
 import pandas as pd
+from scipy.stats import multinomial
 from .io import set_log_level
 
 __all__ = [
     "hypno_str_to_int",
     "hypno_int_to_str",
+    "hypno_consolidate_stages",
     "hypno_upsample_to_sf",
     "hypno_upsample_to_data",
     "hypno_find_periods",
     "load_profusion_hypno",
+    "simulate_hypno",
```

Comment:
Can you please add `simulate_hypno` to docs/api.rst and docs/changelog.rst? No need for `hypno_consolidate_stages` since as you noted this is eventually going to be a method of the class Hypnogram.